app/config/ConfigKeys.scala

```diff
@@ -42,6 +42,7 @@ object ConfigKeys {
   val useLanguageSelectorFeature: String = "features.useLanguageSelector.enabled"
   val submitReturnFeatures: String = "features.submitReturnFeatures.enabled"
   val agentAccessFeature: String = "features.agentAccess.enabled"
+  val changeClientFeature: String = "features.changeClientFeature.enabled"
```

Comment:
Don't need a new feature switch - lump this with the Agent Access feature as its all the same journey